<launch>
  <arg name="language" default="en"/>
  <arg name="intern_file" default="none"/> <!--$(find ontologenius)/file_intern/ontologenius.owl-->
  <arg name="robot_file" default="none"/>
  <arg name="human_file" default="none"/>
  <arg name="config_file" default="none"/>
  <arg name="display" default="false"/>
  <arg name="root" default="none"/>
  <arg name="files" default=""/>
  <arg name="tcmalloc_path" default=""/>

  <env name="LD_PRELOAD" value="$(arg tcmalloc_path)"/>

  <node name="ontologenius_core" pkg="ontologenius" type="ontologenius_multi" output="screen" args="--root $(arg root) -l $(arg language) -c $(arg config_file) -i $(arg intern_file) -d $(arg display) -r $(arg robot_file) -h $(arg human_file) $(arg files)"> </node>
</launch>
